api/proto/external.proto

```diff
@@ -224,10 +227,19 @@ message TxPrefix {
     repeated TxOut outputs = 2;
 
     // Fee paid to the foundation for this transaction
-    uint64 fee = 3;
+    fixed64 fee = 3;
```

Comment:
Is this a breaking change?

Reply:
That doesn't require changing the types or tags of existing fields.

Reply:
I think the organization is much better if the fee is next to the token id, it makes it easier to read and understand. Is there a problem with that?

Reply:
we're going to redesign per discussion so that this isn't a breaking change, and to the extent there is, navigate that via the block version bumping